ci_pod/jupyterlab/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM fedora:38
+
+ARG JUPYTERLAB_VERSION
+ENV JUPYTERLAB_VERSION=$JUPYTERLAB_VERSION
+ARG MAINTAINER
+ENV MAINTAINER=$MAINTAINER
+
+LABEL maintainer="Your Name <$MAINTAINER>"
+
+# Install system dependencies
+RUN dnf update -y && \
+    dnf install -y wget podman && \
+    dnf install -y curl && \
+    dnf install -y python3 python3-pip python3-devel nodejs npm && \
```

Comment:
It won't really matter all that much, but you might as well combine all of the `dnf install` commands.